_data/events/Pune-2.json

```diff
@@ -0,0 +1,19 @@
+{
+    "title": "GDCR17 e-zest, Pune",
+    "url": "http://www.e-zest.com/gdcr-2017",
+    "moderators": [
+        "@christianhujer",
+        "@nelkinda",
+        "@ezest"
```

Comment:
Please move the sponsors to the [`sponsors`](https://github.com/coderetreat/coderetreat.github.io/blob/master/events/event_schema.json#L61-L67) field